plugin/plugin.go

```diff
@@ -154,8 +154,22 @@ func (p *plugin) generateRegexVars(file *generator.FileDescriptor, message *gene
 	ccTypeName := generator.CamelCaseSlice(message.TypeName())
 	for _, field := range message.Field {
 		validator := getFieldValidatorIfAny(field)
-		if validator != nil && validator.Regex != nil {
+		if validator != nil && (validator.Regex != nil || validator.UuidVer != nil) {
```

Comment:
We should print an warning if the field has both a `regex` and a `uuid_ver` field as that would be an error due to the conflicting constraints.

Reply:
I added to print a warning message.
It think it would be better to even fail the generation because the result won't be what the user expects.